File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        n=len(nums)
        arr=[False]*n
        nums.sort()
        for i in range(n):
            arr[nums[i]-1]=True
        res=[]
        for i in range(len(arr)):
            if(arr[i]==False):
                res.append(i+1)
        return res

            
        

        


          

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        int n=nums.size();
        vector<int> res;
        vector <int> v(n+1,0);
        for(int i=0;i<n;i++){
            v[nums[i]]=1;
        }
        for(int i=1;i<n+1;i++){
            if(v[i]==0){
                res.push_back(i);
            }
        }
        return res;
    }
};